pkg/oomstore/join.go

```diff
@@ -58,7 +58,9 @@ func (s *OomStore) ChannelJoin(ctx context.Context, opt types.ChannelJoinOpt) (*
 		if err != nil {
 			return nil, err
 		}
-		revisionRangeMap[groupName] = revisionRanges
+		if len(revisionRanges) > 0 {
```

Comment:
if this group doesn't have any revision, should we return the error? just like `Export`

Reply:
sound good

Reply:
done